### ISSUE TITLE
Accordion & Collapsible Contents

### ISSUE THREAD

**Issue by hiteshah185** [TeraServ/starfish#63]
- [ ] All accordion should have corresponding icon for expanded and collapsed view.
- [ ] Fonts should be hierarchical and same throughout the application
